src/utils/db/mod.rs

```diff
@@ -23,6 +22,9 @@ use tracing::info;
 /// for simplicity. The actual value for other underlying DB might be slightly
 /// different but that is negligible for calculating the total reachable data
 /// size
+// NOTE: For some reason linter detects this as unused, even though it is used
+// below. Perhaps async-trait has something to do with it.
+#[allow(unused)]
```

Comment:
I want some clarity on this matter before we merge this PR.

Reply:
It turns out that neither this constant nor the function was used. I've removed both.